src/main/webapp/src/app/map/map.component.ts

```diff
@@ -28,12 +30,62 @@ export class MapComponent implements AfterViewInit {
   }
 
   mapInitializer(): void {
+    
+    // Coordinates fetched from the user's location to set the center of the map.
     let centerOfMapCoordinates = new google.maps.LatLng(this.location.lat, this.location.lng);
 
     let mapOptions: google.maps.MapOptions = {
       center: centerOfMapCoordinates,
       zoom: 10
     };
     this.map = new google.maps.Map(this.mapContainer.nativeElement, mapOptions);
+
+    //Adding other markers
+     this.loadAllMarkers();
+  }
+
+
+  loadAllMarkers(): void {
+   
+    // const markersJson = [
+    //   {
+    //     position: new google.maps.LatLng( -33.513059, 151.1234),
+    //     map: this.map,
+    //     title: HiddenGemService.name,
+    //   },
+    //   {}
+    // ];
+    const markers = [
+      {
+        position: new google.maps.LatLng( -33.513059, 151.1234),
+        map: this.map,
+        title: "Quay restaurant"
+      },
+      {
+        position: new google.maps.LatLng(-33.47,151.865),
+        map: this.map,
+        title: "Food Van cafe"
+      }
+    ];
+
+    markers.forEach(markerInfo => {
+      // //Creating a new marker object
+      const markerObj = new google.maps.Marker({
+        ...markerInfo
+      });
+
+      //creating a new info window with markers info
+      const infoWindow = new google.maps.InfoWindow({
+        content: markerInfo.title
```

Comment:
If you want, you could already set up the info window with the info of the hidden gems. For example, structuring the info window with the hidden gem's name, address, star rating, etc. So that, later on, it will be easier to integrate the real info with how the markers are created (we will simply need to replace the hardcoded data with real data, without having to work on info windows' layout)
You could maybe use the hiddem gem dummy data to do that? Or create your own hardcoded data.